src/migration/1600969675208-defaultConfigItems.ts

```diff
@@ -0,0 +1,23 @@
+import {MigrationInterface, QueryRunner} from "typeorm";
+
+export class defaultConfigItems1600969675208 implements MigrationInterface {
+
+    public async up(queryRunner: QueryRunner): Promise<any> {
+        await queryRunner.query(
+            'INSERT INTO "configuration" ("key", "value") VALUES ("teamRegistrationActive", "true")', undefined
```

Comment:
@SpencerKaiser thoughts on if these should be true or false?

Reply:
Ya. I was confused about this too. I guess @SpencerKaiser mentioned we need to have default values disabled. So should I change it to false?